tests/phpunit/tests/test-admin.php

```diff
@@ -49,6 +49,25 @@ public function test_admin_bar_menu() {
 		$this->assertEquals( '/wp-admin/edit.php?lang=fr', $fr->href );
 	}
 
+	public function test_admin_bar_menu_should_hide() {
+		global $wp_admin_bar;
+		add_filter( 'show_admin_bar', '__return_true' ); // Make sure to show admin bar.
+
+		$this->go_to( admin_url( 'post-new.php?post_type=page' ) );
+		$GLOBALS['pagenow'] = 'post-new.php';
+		$GLOBALS['typenow'] = 'page';
+
+		$links_model = self::$model->get_links_model();
+		$pll_admin = new PLL_Admin( $links_model );
+		$pll_admin->init();
+
+		_wp_admin_bar_init();
+		do_action_ref_array( 'admin_bar_menu', array( &$wp_admin_bar ) );
+
+		$languages = $wp_admin_bar->get_node( 'languages' );
+		$this->assertEmpty( $languages );
```

Comment:
```suggestion
		$this->assertEmpty( $languages, 'Languages admin bar menu should be hidden on post edit pages' );
```

Reply:
Done in [b9ae37a](https://github.com/polylang/polylang/pull/1698/commits/b9ae37aed1929aee524120a4556f53a2537318c1).